code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -153,9 +152,10 @@
 			suit=/obj/item/clothing/suit/space/eva/plasmaman/nuclear
 			helm=/obj/item/clothing/head/helmet/space/eva/plasmaman/nuclear
 
-	if((H.mind.special_role == SPECIAL_ROLE_WIZARD) || (H.mind.special_role == SPECIAL_ROLE_WIZARD_APPRENTICE))
-		H.equip_to_slot(new /obj/item/clothing/suit/space/eva/plasmaman/wizard(H), slot_wear_suit)
-		H.equip_to_slot(new /obj/item/clothing/head/helmet/space/eva/plasmaman/wizard(H), slot_head)
+	if(H.mind)
```

Comment:
The way this conditional associates, this will mean that only mindless mobs will get suits or headwear. You probably want an `&&` instead.

Reply:
not liking && going wiht plan b.